src/protocol/thread.js

```diff
@@ -1354,7 +1437,7 @@ const ThreadFront = {
       pause.instantiate(message.pauseId, message.data);
       if (message.argumentValues) {
         message.argumentValues = message.argumentValues.map(
-          v => new ValueFront(pause, v)
+          (v) => new ValueFront(pause, v)
```

Comment:
I'll file an issue for adding a prettier config so we keep things consistent and don't add unecessary parens.